server/game/cards/05-IOUF/YunxuJiang.js

```diff
@@ -0,0 +1,18 @@
+const DudeCard = require('../../dudecard.js');
+
+class YunxuJiang extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.isParticipating() && (this.game.shootout.shootoutLocation.controller === this.controller || this.getGameLocation().isHome(this.controller)),
```

Comment:
`this.game.shootout.shootoutLocation` is of type `GameLocation` and as such does not have controller. you need to get to the card (`this.game.shootout.shootoutLocation.locationCard.controller`)